apb.yml

```diff
@@ -1,6 +1,6 @@
 version: 1.0
 name: keycloak-apb
-description: Keycloak - Open Source Identity and Access Management
+description: Keycloak - AEROGEAR-1993
```

Comment:
Should probably change back the description to its original content.

Reply:
This is only for debug/validation part

Reply:
Done!